### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by NapatAomsinS**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by NapatAomsinS — COMMENTED** [review]
A few more words is added to game over screen such that user know how to restart the game.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (NapatAomsinS) [inline index.html:51]: Maybe it is better to add a button instead ?

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
```
Comment (Napat-Osaka) [inline index.html]: `<h2 hidden>Test suggestion</h2>`

**Review by Napat-Osaka — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Napat-Osaka) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again 🤖!</h2>
```

**Pull request merged**